eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/publisher/impl/AlertPublisherImpl.java

```diff
@@ -73,11 +76,11 @@ public void nextEvent(PublishPartition partition, AlertStreamEvent event) {
     private void notifyAlert(PublishPartition partition, AlertStreamEvent event) {
         // remove the column values for publish plugin match
         partition.getColumnValues().clear();
-        if (!publishPluginMapping.containsKey(partition)) {
+        if (!publishPluginMapping.containsKey(partition.getPublishId())) {
```

Comment:
Sorry for the late reply. 

For the alert dropping off of unwanted stream is handled in AlertBoltOutputCollectorWrapper, which are kept as it is. 

In this class, i just simplify the coding, which should keeps the original logic. How do you think? @garrettlish